src/fpm/manifest/package.f90

```diff
@@ -132,6 +134,9 @@ subroutine new_package(self, table, root, error)
            return
         end if
 
+        self%implicit_none = .false.
+        call get_value(table, "implicit_none", self%implicit_none)
```

Comment:
The fetched value is `intent(out)`, to pass the default just add the default optional argument like below:
```suggestion
        call get_value(table, "implicit_none", self%implicit_none, .false.)
```

Reply:
Perfect, thanks!